demo1/StaticLib/ClassMatrixs.cpp

```
@@ -3,10 +3,44 @@
 
 using namespace std;
 
-Matrixs::Matrixs() {
+#pragma region exceptions
```

I decided to move all checks repetitions into separate methods to follow DRY, as well as added exceptions throwing instead of just printing errors into console